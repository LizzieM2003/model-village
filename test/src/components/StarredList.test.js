import React from 'react';

import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import { StarredList } from './StarredList';
import starredData from '../data/starred_influencers.json';

import { ENGAGEMENT, DESC } from '../actions/constants';

configure({ adapter: new Adapter() });

describe('StarredList', () => {
  const mockSortStarred = jest.fn();

  const props = {
    starred: starredData,
    sortStarred: mockSortStarred
  };

  const starredList = shallow(<StarredList {...props} />);

  it('renders correctly', () => {
    expect(starredList).toMatchSnapshot();
  });

  it('has one StarredItem component for each starred influencer', () => {
    expect(starredList.find('.StarredList-items').children().length).toBe(
      starredData.length
    );
  });

  it('contains a sort by dropdown', () => {
    expect(starredList.find('.StarredList__dropdown').exists()).toBe(true);
  });

  it('changes the sort order when select dropdown option', () => {
    starredList
      .find('.StarredList__dropdown')
      .simulate('change', { target: { value: `${ENGAGEMENT} ${DESC}` } });
    expect(mockSortStarred).toHaveBeenCalled();
  });
});
