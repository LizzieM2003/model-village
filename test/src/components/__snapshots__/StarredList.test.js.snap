// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StarredList renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <StarredList
    sortStarred={[Function]}
    starred={
        Array [
            Object {
              "influencer_full_name": "Anna Griffin",
              "influencer_id": 987,
              "influencer_instagram_profile_image": "https://randomuser.me/api/portraits/women/6.jpg",
              "influencer_instagram_username": "anna_griffin_123",
              "statistics": Object {
                "engagement": "6.84",
                "followers": 21231,
              },
            },
            Object {
              "influencer_full_name": "Melissa Peterson",
              "influencer_id": 876,
              "influencer_instagram_profile_image": "https://randomuser.me/api/portraits/women/84.jpg",
              "influencer_instagram_username": "melissa_peterson",
              "statistics": Object {
                "engagement": "5.23",
                "followers": 5122,
              },
            },
            Object {
              "influencer_full_name": "Kathryn Smith",
              "influencer_id": 765,
              "influencer_instagram_profile_image": "https://randomuser.me/api/portraits/women/71.jpg",
              "influencer_instagram_username": "kat_smith",
              "statistics": Object {
                "engagement": "2.36",
                "followers": 7161,
              },
            },
            Object {
              "influencer_full_name": "Patricia Rios",
              "influencer_id": 654,
              "influencer_instagram_profile_image": "https://randomuser.me/api/portraits/women/49.jpg",
              "influencer_instagram_username": "patricia_rios_1993",
              "statistics": Object {
                "engagement": "1.61",
                "followers": 91100,
              },
            },
            Object {
              "influencer_full_name": "Joan Robertson",
              "influencer_id": 543,
              "influencer_instagram_profile_image": "https://randomuser.me/api/portraits/women/26.jpg",
              "influencer_instagram_username": "j_robertson_lol",
              "statistics": Object {
                "engagement": "6.28",
                "followers": 16013,
              },
            },
            Object {
              "influencer_full_name": "Sharon Hopkins",
              "influencer_id": 432,
              "influencer_instagram_profile_image": "https://randomuser.me/api/portraits/women/44.jpg",
              "influencer_instagram_username": "sharon_hopkins234",
              "statistics": Object {
                "engagement": "4.73",
                "followers": 37300,
              },
            },
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="StarredList__header"
>
          <h2>
                    Starred Influencers
          </h2>
          <select
                    className="StarredList__dropdown"
                    onChange={[Function]}
          >
                    <option
                              disabled={true}
                              selected={true}
                              value=""
                    >
                              Sort by
                    </option>
                    <option
                              value="FOLLOWERS ASC"
                    >
                              Followers (Low to High)
                    </option>
                    <option
                              value="FOLLOWERS DESC"
                    >
                              Followers (High to Low)
                    </option>
                    <option
                              value="ENGAGEMENT ASC"
                    >
                              Engagement (Low to High)
                    </option>
                    <option
                              value="ENGAGEMENT DESC"
                    >
                              Engagement (High to Low)
                    </option>
          </select>
</div>,
        <div
          className="StarredList-items"
>
          <Connect(Component)
                    influencer_full_name="Anna Griffin"
                    influencer_id={987}
                    influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/6.jpg"
                    influencer_instagram_username="anna_griffin_123"
                    statistics={
                              Object {
                                        "engagement": "6.84",
                                        "followers": 21231,
                                      }
                    }
          />
          <Connect(Component)
                    influencer_full_name="Melissa Peterson"
                    influencer_id={876}
                    influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/84.jpg"
                    influencer_instagram_username="melissa_peterson"
                    statistics={
                              Object {
                                        "engagement": "5.23",
                                        "followers": 5122,
                                      }
                    }
          />
          <Connect(Component)
                    influencer_full_name="Kathryn Smith"
                    influencer_id={765}
                    influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/71.jpg"
                    influencer_instagram_username="kat_smith"
                    statistics={
                              Object {
                                        "engagement": "2.36",
                                        "followers": 7161,
                                      }
                    }
          />
          <Connect(Component)
                    influencer_full_name="Patricia Rios"
                    influencer_id={654}
                    influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/49.jpg"
                    influencer_instagram_username="patricia_rios_1993"
                    statistics={
                              Object {
                                        "engagement": "1.61",
                                        "followers": 91100,
                                      }
                    }
          />
          <Connect(Component)
                    influencer_full_name="Joan Robertson"
                    influencer_id={543}
                    influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/26.jpg"
                    influencer_instagram_username="j_robertson_lol"
                    statistics={
                              Object {
                                        "engagement": "6.28",
                                        "followers": 16013,
                                      }
                    }
          />
          <Connect(Component)
                    influencer_full_name="Sharon Hopkins"
                    influencer_id={432}
                    influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/44.jpg"
                    influencer_instagram_username="sharon_hopkins234"
                    statistics={
                              Object {
                                        "engagement": "4.73",
                                        "followers": 37300,
                                      }
                    }
          />
</div>,
      ],
      "className": "StarredList",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h2>
              Starred Influencers
</h2>,
            <select
              className="StarredList__dropdown"
              onChange={[Function]}
>
              <option
                            disabled={true}
                            selected={true}
                            value=""
              >
                            Sort by
              </option>
              <option
                            value="FOLLOWERS ASC"
              >
                            Followers (Low to High)
              </option>
              <option
                            value="FOLLOWERS DESC"
              >
                            Followers (High to Low)
              </option>
              <option
                            value="ENGAGEMENT ASC"
              >
                            Engagement (Low to High)
              </option>
              <option
                            value="ENGAGEMENT DESC"
              >
                            Engagement (High to Low)
              </option>
</select>,
          ],
          "className": "StarredList__header",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Starred Influencers",
            },
            "ref": null,
            "rendered": "Starred Influencers",
            "type": "h2",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <option
                  disabled={true}
                  selected={true}
                  value=""
>
                  Sort by
</option>,
                <option
                  value="FOLLOWERS ASC"
>
                  Followers (Low to High)
</option>,
                <option
                  value="FOLLOWERS DESC"
>
                  Followers (High to Low)
</option>,
                <option
                  value="ENGAGEMENT ASC"
>
                  Engagement (Low to High)
</option>,
                <option
                  value="ENGAGEMENT DESC"
>
                  Engagement (High to Low)
</option>,
              ],
              "className": "StarredList__dropdown",
              "onChange": [Function],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Sort by",
                  "disabled": true,
                  "selected": true,
                  "value": "",
                },
                "ref": null,
                "rendered": "Sort by",
                "type": "option",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Followers (Low to High)",
                  "value": "FOLLOWERS ASC",
                },
                "ref": null,
                "rendered": "Followers (Low to High)",
                "type": "option",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Followers (High to Low)",
                  "value": "FOLLOWERS DESC",
                },
                "ref": null,
                "rendered": "Followers (High to Low)",
                "type": "option",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Engagement (Low to High)",
                  "value": "ENGAGEMENT ASC",
                },
                "ref": null,
                "rendered": "Engagement (Low to High)",
                "type": "option",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Engagement (High to Low)",
                  "value": "ENGAGEMENT DESC",
                },
                "ref": null,
                "rendered": "Engagement (High to Low)",
                "type": "option",
              },
            ],
            "type": "select",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <Connect(Component)
              influencer_full_name="Anna Griffin"
              influencer_id={987}
              influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/6.jpg"
              influencer_instagram_username="anna_griffin_123"
              statistics={
                            Object {
                                          "engagement": "6.84",
                                          "followers": 21231,
                                        }
              }
/>,
            <Connect(Component)
              influencer_full_name="Melissa Peterson"
              influencer_id={876}
              influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/84.jpg"
              influencer_instagram_username="melissa_peterson"
              statistics={
                            Object {
                                          "engagement": "5.23",
                                          "followers": 5122,
                                        }
              }
/>,
            <Connect(Component)
              influencer_full_name="Kathryn Smith"
              influencer_id={765}
              influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/71.jpg"
              influencer_instagram_username="kat_smith"
              statistics={
                            Object {
                                          "engagement": "2.36",
                                          "followers": 7161,
                                        }
              }
/>,
            <Connect(Component)
              influencer_full_name="Patricia Rios"
              influencer_id={654}
              influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/49.jpg"
              influencer_instagram_username="patricia_rios_1993"
              statistics={
                            Object {
                                          "engagement": "1.61",
                                          "followers": 91100,
                                        }
              }
/>,
            <Connect(Component)
              influencer_full_name="Joan Robertson"
              influencer_id={543}
              influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/26.jpg"
              influencer_instagram_username="j_robertson_lol"
              statistics={
                            Object {
                                          "engagement": "6.28",
                                          "followers": 16013,
                                        }
              }
/>,
            <Connect(Component)
              influencer_full_name="Sharon Hopkins"
              influencer_id={432}
              influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/44.jpg"
              influencer_instagram_username="sharon_hopkins234"
              statistics={
                            Object {
                                          "engagement": "4.73",
                                          "followers": 37300,
                                        }
              }
/>,
          ],
          "className": "StarredList-items",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": "987",
            "nodeType": "class",
            "props": Object {
              "influencer_full_name": "Anna Griffin",
              "influencer_id": 987,
              "influencer_instagram_profile_image": "https://randomuser.me/api/portraits/women/6.jpg",
              "influencer_instagram_username": "anna_griffin_123",
              "statistics": Object {
                "engagement": "6.84",
                "followers": 21231,
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "876",
            "nodeType": "class",
            "props": Object {
              "influencer_full_name": "Melissa Peterson",
              "influencer_id": 876,
              "influencer_instagram_profile_image": "https://randomuser.me/api/portraits/women/84.jpg",
              "influencer_instagram_username": "melissa_peterson",
              "statistics": Object {
                "engagement": "5.23",
                "followers": 5122,
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "765",
            "nodeType": "class",
            "props": Object {
              "influencer_full_name": "Kathryn Smith",
              "influencer_id": 765,
              "influencer_instagram_profile_image": "https://randomuser.me/api/portraits/women/71.jpg",
              "influencer_instagram_username": "kat_smith",
              "statistics": Object {
                "engagement": "2.36",
                "followers": 7161,
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "654",
            "nodeType": "class",
            "props": Object {
              "influencer_full_name": "Patricia Rios",
              "influencer_id": 654,
              "influencer_instagram_profile_image": "https://randomuser.me/api/portraits/women/49.jpg",
              "influencer_instagram_username": "patricia_rios_1993",
              "statistics": Object {
                "engagement": "1.61",
                "followers": 91100,
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "543",
            "nodeType": "class",
            "props": Object {
              "influencer_full_name": "Joan Robertson",
              "influencer_id": 543,
              "influencer_instagram_profile_image": "https://randomuser.me/api/portraits/women/26.jpg",
              "influencer_instagram_username": "j_robertson_lol",
              "statistics": Object {
                "engagement": "6.28",
                "followers": 16013,
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "432",
            "nodeType": "class",
            "props": Object {
              "influencer_full_name": "Sharon Hopkins",
              "influencer_id": 432,
              "influencer_instagram_profile_image": "https://randomuser.me/api/portraits/women/44.jpg",
              "influencer_instagram_username": "sharon_hopkins234",
              "statistics": Object {
                "engagement": "4.73",
                "followers": 37300,
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="StarredList__header"
>
            <h2>
                        Starred Influencers
            </h2>
            <select
                        className="StarredList__dropdown"
                        onChange={[Function]}
            >
                        <option
                                    disabled={true}
                                    selected={true}
                                    value=""
                        >
                                    Sort by
                        </option>
                        <option
                                    value="FOLLOWERS ASC"
                        >
                                    Followers (Low to High)
                        </option>
                        <option
                                    value="FOLLOWERS DESC"
                        >
                                    Followers (High to Low)
                        </option>
                        <option
                                    value="ENGAGEMENT ASC"
                        >
                                    Engagement (Low to High)
                        </option>
                        <option
                                    value="ENGAGEMENT DESC"
                        >
                                    Engagement (High to Low)
                        </option>
            </select>
</div>,
          <div
            className="StarredList-items"
>
            <Connect(Component)
                        influencer_full_name="Anna Griffin"
                        influencer_id={987}
                        influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/6.jpg"
                        influencer_instagram_username="anna_griffin_123"
                        statistics={
                                    Object {
                                                "engagement": "6.84",
                                                "followers": 21231,
                                              }
                        }
            />
            <Connect(Component)
                        influencer_full_name="Melissa Peterson"
                        influencer_id={876}
                        influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/84.jpg"
                        influencer_instagram_username="melissa_peterson"
                        statistics={
                                    Object {
                                                "engagement": "5.23",
                                                "followers": 5122,
                                              }
                        }
            />
            <Connect(Component)
                        influencer_full_name="Kathryn Smith"
                        influencer_id={765}
                        influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/71.jpg"
                        influencer_instagram_username="kat_smith"
                        statistics={
                                    Object {
                                                "engagement": "2.36",
                                                "followers": 7161,
                                              }
                        }
            />
            <Connect(Component)
                        influencer_full_name="Patricia Rios"
                        influencer_id={654}
                        influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/49.jpg"
                        influencer_instagram_username="patricia_rios_1993"
                        statistics={
                                    Object {
                                                "engagement": "1.61",
                                                "followers": 91100,
                                              }
                        }
            />
            <Connect(Component)
                        influencer_full_name="Joan Robertson"
                        influencer_id={543}
                        influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/26.jpg"
                        influencer_instagram_username="j_robertson_lol"
                        statistics={
                                    Object {
                                                "engagement": "6.28",
                                                "followers": 16013,
                                              }
                        }
            />
            <Connect(Component)
                        influencer_full_name="Sharon Hopkins"
                        influencer_id={432}
                        influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/44.jpg"
                        influencer_instagram_username="sharon_hopkins234"
                        statistics={
                                    Object {
                                                "engagement": "4.73",
                                                "followers": 37300,
                                              }
                        }
            />
</div>,
        ],
        "className": "StarredList",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h2>
                Starred Influencers
</h2>,
              <select
                className="StarredList__dropdown"
                onChange={[Function]}
>
                <option
                                disabled={true}
                                selected={true}
                                value=""
                >
                                Sort by
                </option>
                <option
                                value="FOLLOWERS ASC"
                >
                                Followers (Low to High)
                </option>
                <option
                                value="FOLLOWERS DESC"
                >
                                Followers (High to Low)
                </option>
                <option
                                value="ENGAGEMENT ASC"
                >
                                Engagement (Low to High)
                </option>
                <option
                                value="ENGAGEMENT DESC"
                >
                                Engagement (High to Low)
                </option>
</select>,
            ],
            "className": "StarredList__header",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Starred Influencers",
              },
              "ref": null,
              "rendered": "Starred Influencers",
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <option
                    disabled={true}
                    selected={true}
                    value=""
>
                    Sort by
</option>,
                  <option
                    value="FOLLOWERS ASC"
>
                    Followers (Low to High)
</option>,
                  <option
                    value="FOLLOWERS DESC"
>
                    Followers (High to Low)
</option>,
                  <option
                    value="ENGAGEMENT ASC"
>
                    Engagement (Low to High)
</option>,
                  <option
                    value="ENGAGEMENT DESC"
>
                    Engagement (High to Low)
</option>,
                ],
                "className": "StarredList__dropdown",
                "onChange": [Function],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Sort by",
                    "disabled": true,
                    "selected": true,
                    "value": "",
                  },
                  "ref": null,
                  "rendered": "Sort by",
                  "type": "option",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Followers (Low to High)",
                    "value": "FOLLOWERS ASC",
                  },
                  "ref": null,
                  "rendered": "Followers (Low to High)",
                  "type": "option",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Followers (High to Low)",
                    "value": "FOLLOWERS DESC",
                  },
                  "ref": null,
                  "rendered": "Followers (High to Low)",
                  "type": "option",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Engagement (Low to High)",
                    "value": "ENGAGEMENT ASC",
                  },
                  "ref": null,
                  "rendered": "Engagement (Low to High)",
                  "type": "option",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Engagement (High to Low)",
                    "value": "ENGAGEMENT DESC",
                  },
                  "ref": null,
                  "rendered": "Engagement (High to Low)",
                  "type": "option",
                },
              ],
              "type": "select",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <Connect(Component)
                influencer_full_name="Anna Griffin"
                influencer_id={987}
                influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/6.jpg"
                influencer_instagram_username="anna_griffin_123"
                statistics={
                                Object {
                                                "engagement": "6.84",
                                                "followers": 21231,
                                              }
                }
/>,
              <Connect(Component)
                influencer_full_name="Melissa Peterson"
                influencer_id={876}
                influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/84.jpg"
                influencer_instagram_username="melissa_peterson"
                statistics={
                                Object {
                                                "engagement": "5.23",
                                                "followers": 5122,
                                              }
                }
/>,
              <Connect(Component)
                influencer_full_name="Kathryn Smith"
                influencer_id={765}
                influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/71.jpg"
                influencer_instagram_username="kat_smith"
                statistics={
                                Object {
                                                "engagement": "2.36",
                                                "followers": 7161,
                                              }
                }
/>,
              <Connect(Component)
                influencer_full_name="Patricia Rios"
                influencer_id={654}
                influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/49.jpg"
                influencer_instagram_username="patricia_rios_1993"
                statistics={
                                Object {
                                                "engagement": "1.61",
                                                "followers": 91100,
                                              }
                }
/>,
              <Connect(Component)
                influencer_full_name="Joan Robertson"
                influencer_id={543}
                influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/26.jpg"
                influencer_instagram_username="j_robertson_lol"
                statistics={
                                Object {
                                                "engagement": "6.28",
                                                "followers": 16013,
                                              }
                }
/>,
              <Connect(Component)
                influencer_full_name="Sharon Hopkins"
                influencer_id={432}
                influencer_instagram_profile_image="https://randomuser.me/api/portraits/women/44.jpg"
                influencer_instagram_username="sharon_hopkins234"
                statistics={
                                Object {
                                                "engagement": "4.73",
                                                "followers": 37300,
                                              }
                }
/>,
            ],
            "className": "StarredList-items",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": "987",
              "nodeType": "class",
              "props": Object {
                "influencer_full_name": "Anna Griffin",
                "influencer_id": 987,
                "influencer_instagram_profile_image": "https://randomuser.me/api/portraits/women/6.jpg",
                "influencer_instagram_username": "anna_griffin_123",
                "statistics": Object {
                  "engagement": "6.84",
                  "followers": 21231,
                },
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "876",
              "nodeType": "class",
              "props": Object {
                "influencer_full_name": "Melissa Peterson",
                "influencer_id": 876,
                "influencer_instagram_profile_image": "https://randomuser.me/api/portraits/women/84.jpg",
                "influencer_instagram_username": "melissa_peterson",
                "statistics": Object {
                  "engagement": "5.23",
                  "followers": 5122,
                },
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "765",
              "nodeType": "class",
              "props": Object {
                "influencer_full_name": "Kathryn Smith",
                "influencer_id": 765,
                "influencer_instagram_profile_image": "https://randomuser.me/api/portraits/women/71.jpg",
                "influencer_instagram_username": "kat_smith",
                "statistics": Object {
                  "engagement": "2.36",
                  "followers": 7161,
                },
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "654",
              "nodeType": "class",
              "props": Object {
                "influencer_full_name": "Patricia Rios",
                "influencer_id": 654,
                "influencer_instagram_profile_image": "https://randomuser.me/api/portraits/women/49.jpg",
                "influencer_instagram_username": "patricia_rios_1993",
                "statistics": Object {
                  "engagement": "1.61",
                  "followers": 91100,
                },
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "543",
              "nodeType": "class",
              "props": Object {
                "influencer_full_name": "Joan Robertson",
                "influencer_id": 543,
                "influencer_instagram_profile_image": "https://randomuser.me/api/portraits/women/26.jpg",
                "influencer_instagram_username": "j_robertson_lol",
                "statistics": Object {
                  "engagement": "6.28",
                  "followers": 16013,
                },
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "432",
              "nodeType": "class",
              "props": Object {
                "influencer_full_name": "Sharon Hopkins",
                "influencer_id": 432,
                "influencer_instagram_profile_image": "https://randomuser.me/api/portraits/women/44.jpg",
                "influencer_instagram_username": "sharon_hopkins234",
                "statistics": Object {
                  "engagement": "4.73",
                  "followers": 37300,
                },
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
